releasenotes/notes/feat-debugging-py311-support-2160a5590048c7c3.yaml

```diff
@@ -0,0 +1,3 @@
+features:
+  - |
+    dynamic instrumentation: Added Python 3.11 support to Dynamic Instrumentation.
```

Comment:
```suggestion
    dynamic instrumentation: Add Python 3.11 support.
```

Reply:
I don't agree with the above suggestion. This could be interpreted as something actionable on the user, i.e. the user needs to `add python 3.11 support`. If anything, our [release note guidelines](https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2715353865/Release+Note+Guidelines#Style-Guidelines) suggest using third person present tense, i.e.
```
dynamic instrumentation: This change adds Python 3.11 support to Dynamic Instrumentation.
```